uaclient/cli.py

```diff
@@ -511,7 +511,12 @@ def wrapper(*args, **kwargs):
             sys.exit(1)
         except util.UrlError as exc:
             with util.disable_log_to_console():
-                logging.exception(exc)
+                msg_args = {"url": getattr(exc, "url", None), "error": exc}
```

Comment:
Either this doesn't need to use `getattr` or the next line _does_. Looking at `util.UrlError`, we will always set `self.url` in `__init__`, so we can just make this `exc.url`.

Reply:
riiight. good one.